.github/workflows/build.yaml

```diff
@@ -27,10 +27,8 @@ jobs:
             TAG_VERSION: "22.13"
           - BASE_IMAGE: "node:22.12-bookworm-slim"
             TAG_VERSION: "22.12"
-          - BASE_IMAGE: "node:20.17-bookworm-slim"
```

Comment:
I think this is still used in some places, so I would rather just add the 20.18 version as extra entry